### PR TITLE
Update generate-certs.sh

### PR DESCRIPTION
**Pull request by aria1991**
By default, your script generates a certificate with a 2048-bit RSA key and an SHA-256 digest algorithm. Consider increasing the key size to 4096 bits, and also use the Elliptic Curve Digital Signature Algorithm (ECDSA) or EdDSA over RSA key.

### CODE REVIEW

**Comment by slawlor** [comment]
I don't see how it could cause a problem, however you validated that HAProxy starts and runs with this change? Checking the :443 for the ability to still forward connections?

**Comment by aria1991** [comment]
> I don't see how it could cause a problem, however you validated that HAProxy starts and runs with this change? Checking the :443 for the ability to still forward connections?

@slawlor my test passed successfully!

**Review by slawlor — APPROVED** [review]

**Comment by delawar77** [comment]
NAME	IP	COUNTRY	RESPONSE TIME
GardenSnakes Proxy #1	141.148.228.97:82	🇳🇱	3ms
GardenSnakes Proxy #2	68.183.8.87:80	🇳🇱	4ms
GardenSnakes Proxy #3	64.227.75.24:82	🇳🇱	9ms
Jaiden051's Proxy	51.14.190.56:5222	🇬🇧	4ms
GardenSnakes Proxy #4	193.123.34.92:443	🇳🇱	3ms
GardenSnakes Proxy #5	158.101.220.0:443	🇳🇱	5ms
Capsload2's USA Proxy	107.191.125.101:443	🇺🇸	3ms
GardenSnakes Proxy #6	34.105.83.231:443	🇺🇸	2ms
Gary-RR's Proxy	141.148.69.176:443	🇺🇸	2ms

File: proxy/src/generate-certs.sh [hunk 1]
```diff
@@ -19,7 +19,7 @@ export SSL_CONFIG="openssl.cnf"
 export SSL_KEY="key.pem"
 export SSL_CSR="key.csr"
 export SSL_CERT="cert.pem"
-export SSL_SIZE="2048"
+export SSL_SIZE="4096"
```
Comment (Everluis58) [inline proxy/src/generate-certs.sh:22]: bueno

Comment (7koma95) [reply proxy/src/generate-certs.sh:22]: > bueno

**Comment by Everluis58** [comment]
odatos.telcel.com

File: proxy/src/generate-certs.sh (hunk 1 above)
Comment (7koma95) [reply proxy/src/generate-certs.sh:22]: > bueno

**Comment by Everluis58** [comment]
Accepted.

odatos.telcel.com

**Review by 7koma95 — APPROVED** [review]

**Comment by Everluis58** [comment]
He compartido un elemento contigo:

Documento de Google Keep
https://docs.google.com/document/d/13S04zKvkr8NXBvmqTizjL2M7y9LH3G_AL9zKPeqpK4M/edit?usp=sharing&invite=CMuuvXY&ts=64291d3d

No es un archivo adjunto; está almacenado en línea. Para abrir este  
elemento, haz clic en el vínculo de arriba.

**Comment by Everluis58** [comment]
He compartido un elemento contigo:

Documento de Google Keep
https://docs.google.com/document/d/13S04zKvkr8NXBvmqTizjL2M7y9LH3G_AL9zKPeqpK4M/edit?usp=sharing&invite=CMuuvXY&ts=64291d3a

No es un archivo adjunto; está almacenado en línea. Para abrir este  
elemento, haz clic en el vínculo de arriba.

**Comment by Everluis58** [comment]
He compartido un elemento contigo:

Documento de Google Keep
https://docs.google.com/document/d/13S04zKvkr8NXBvmqTizjL2M7y9LH3G_AL9zKPeqpK4M/edit?usp=sharing&invite=CMuuvXY&ts=64291d44

No es un archivo adjunto; está almacenado en línea. Para abrir este  
elemento, haz clic en el vínculo de arriba.

File: proxy/src/generate-certs.sh (hunk 1 above)
Comment (Everluis58) [reply proxy/src/generate-certs.sh:22]: opendns@ssh.mx

Comment (Everluis58) [reply proxy/src/generate-certs.sh:22]: > opendns@ssh.mx

- [ ] Project

**Pull request merged**